check/test.go

```diff
@@ -83,20 +85,42 @@ func (t testItem) flagValue() string {
 		return t.Path
 	}
 
+	if t.Env != "" {
+		return fmt.Sprintf("%s or %s", t.Flag, t.Env)
+	}
+
 	return t.Flag
 }
 
-func (t testItem) findValue(s string) (match bool, value string, err error) {
+func (t testItem) findEnv(s string) (match bool, value string) {
```

Comment:
I'd prefer we use the same pattern for finding values from Env as we currently do for Audit (command line parameters) and AuditConfig (JSON settings). Much as we currently have `pathTestItem` and `flagTestItem` we would have a third option `envTestItem`.  We'd still have a function `t.findValue(s string)` and there would be fewer changes (possibly no changes) in `t.evaluate()`